docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -303,9 +303,68 @@ Continue configuring your Kubernetes provider by clicking **Add Region** and com
     ```yml
     istioCompatibility: enabled: true
     ```
+    
+  - Overrides to publish Node-IP as the server broadcast address.
+  
+    By default, master and T-Server pod IPs are published as the server broadcast address. To publish the IPs of the nodes on which database pods are deployed, add the following YAML to each zone override configuration:
+  
+    ```yml
+    master:
+      podAnnotations:
+        prometheus.io/path: "/prometheus-metrics"
+        # Any other required annotations
+    
+    tserver:
+      extraEnv:
+      - name: NODE_IP
+        valueFrom:
+          fieldRef:
+            fieldPath: status.hostIP
+      serverBroadcastAddress: "$(NODE_IP)"
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - "yb-tserver"
+            topologyKey: kubernetes.io/hostname
+      podAnnotations:
+        prometheus.io/path: "/prometheus-metrics"
```

Comment:
This `prometheus.io/path: "/prometheus-metrics"` annotation seems unrelated. Why is it included in the example? Otherwise, LGTM

Reply:
@bhavin192 Could you please, let me know if I can delete
    ```
  podAnnotations:
        prometheus.io/path: "/prometheus-metrics"
```

Reply:
Yes, we can remove it Jason is correct.

Reply:
Yes, we can remove it Jason is correct.